tests/snmp/test_snmp_memory.py

```diff
@@ -22,37 +22,37 @@ def get_parameter(request):
     return percent
 
 @pytest.fixture()
-def load_memory(duthosts, rand_one_dut_hostname):
+def load_memory(duthosts, enum_rand_one_per_hwsku_hostname):
     """
     Execute script in background to load memory
     """
-    duthost = duthosts[rand_one_dut_hostname]
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     duthost.copy(src='snmp/memory.py', dest='/tmp/memory.py')
     duthost.shell("nohup python /tmp/memory.py > /dev/null 2>&1 &")
     yield
     duthost.shell("killall python /tmp/memory.py", module_ignore_errors=True)
 
-def collect_memory(duthosts, rand_one_dut_hostname):
+def collect_memory(duthosts, enum_rand_one_per_hwsku_hostname):
     """
     Collect memory data from DUT
     """
-    duthost = duthosts[rand_one_dut_hostname]
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     facts = {}
     output = duthost.shell("cat /proc/meminfo")['stdout_lines']
     for line in output:
         split = line.split()
         facts.update({split[0].replace(":", ""): split[-2]})
     return facts
 
-def test_snmp_memory(duthosts, rand_one_dut_hostname, localhost, creds):
+def test_snmp_memory(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds):
     """
     Verify if memory MIB equals to data collected from DUT
     """
-    duthost = duthosts[rand_one_dut_hostname]
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     host_ip = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars['ansible_host']
     snmp_facts = localhost.snmp_facts(host=host_ip, version="v2c",
                                       community=creds["snmp_rocommunity"])['ansible_facts']
-    facts = collect_memory(duthosts, rand_one_dut_hostname)
+    facts = collect_memory(duthosts, enum_rand_one_per_hwsku_hostname)
```

Comment:
Suggest to change the arguments definition of `collect_memory` to:
```
def collect_memory(duthost):
```
Then it would be unnecessary to get duthost using code like `duthost = duthosts[enum_rand_one_per_hwsku_hostname]`.
And the code here can be just:
```
facts = collect_memory(duthost)
```